src/CalcViewModel/UnitConverterViewModel.cpp

```diff
@@ -280,10 +282,33 @@ String ^ UnitConverterViewModel::ConvertToLocalizedString(const std::wstring& st
         return result;
     }
 
+    CurrencyFormatter ^ currencyFormatter;
+
+    switch (cfp)
+    {
+    case CurrencyFormatterParameter::Default:
```

Comment:
Might be better to just make this the 'default:' case

Reply:
Sure.